packages/3dom/demo/index.html

```diff
@@ -69,129 +108,18 @@
         console.log('Changing color to:', event.data.payload);
         material.pbrMetallicRoughness.setBaseColorFactor(event.data.payload);
         break;
+      case 'change-texture':
+        console.log('NEW TEXTURE URI', event.data.payload);
+        model.materials[0].pbrMetallicRoughness.baseColorTexture.texture.source.setURI(event.data.payload);
+        break;
```

Comment:
Doesn't have to happen right now, but we should definitely update this demo to also be able to change the normal and emissive textures. Also, I wonder if we should note in our API that metallicRoughness usually also includes the occlusion channel? Maybe just a comment somewhere.